docs/install.md

```diff
@@ -72,6 +82,15 @@ sure you have [configured `kubectl` to access your cluster](https://kubernetes.i
 1. Confirm that your deployment is launching by running `kubectl get pods`. If pods get stuck in `Pending` status, run
    `kubectl get pv` to check if the necessary volumes have been provisioned (you should see at least 4). Google Cloud
    Platform users may need to [request an increase in storage quota](https://cloud.google.com/compute/quotas).
+   If you have enough volumes, but pods are still stuck, `kubectl cluster-info dump` (very verbose, dump it to a file)
+   may have more information; for instance:
+   ```
+    "Reason": "FailedScheduling",
```

Comment:
This seems like it's addressing a very specific issue that may not come up for many installers. Could you move it to a separate "Troubleshooting" section at the end of the document?

Also, as this documentation is public-facing, could we be stricter on grammar, punctuation, and writing style. For instance,


You can run `kubectl cluster-info dump > dump.txt` to obtain a verbose dump of the cluster state, which can be helpful for debugging issues. Look for warning messages like the following:

```
"Reason": "FailedScheduling",
"Message": "0/3 nodes are available: 1 Insufficient memory, 3 Insufficient cpu.",
```

Reply:
good catch, and yes, this belongs in the troubleshooting document, which already exists as a separate document; the troubleshooting "section" in this one is just a link there. Which is reasonable.